apps/modernization-ui/src/apps/page-builder/components/CreateQuestion/CreateQuestion.tsx

```diff
@@ -350,528 +270,447 @@ export const CreateQuestion = ({ modalRef, question }: any) => {
+                                    />
+                                </>
+                            )}
+                        />
+                        <Controller
+                            control={control}
+                            name="messageVariableId"
+                            rules={{
+                                required: { value: true, message: 'Message ID required' },
+                                pattern: { value: handleValidation(false), message: 'Message ID invalid' }
+                            }}
+                            render={({ field: { onChange, value }, fieldState: { error } }) => (
+                                <Input
+                                    onChange={onChange}
+                                    defaultValue={value}
+                                    label="Message ID"
+                                    type="text"
+                                    required
+                                    error={error?.message}
+                                />
+                            )}
+                        />
+                        <Controller
+                            control={control}
+                            name="messageLabel"
+                            rules={{
+                                required: { value: true, message: 'Message label required' },
+                                pattern: { value: handleValidation(false), message: 'Message label invalid' }
+                            }}
+                            render={({ field: { onChange, value }, fieldState: { error } }) => (
+                                <Input
+                                    onChange={onChange}
+                                    defaultValue={value!}
+                                    label="Message label"
+                                    type="text"
+                                    error={error?.message}
+                                    required
+                                />
+                            )}
+                        />
+                        <Controller
+                            control={control}
+                            name="codeSystem"
+                            render={({ field: { onChange, value } }) => (
+                                <SelectInput
+                                    label="Code system name"
+                                    defaultValue={value}
+                                    onChange={onChange}
+                                    required
+                                    options={codeSystemOptionList.map(({ label }) => {
+                                        return {
+                                            name: label!,
+                                            value: label!
+                                        };
+                                    })}
+                                />
+                            )}
+                        />
```

Comment:
Issue (blocking): `Message Id`, `Message label`, `Code system name`, `HL7 data type` fields should be disabled if `included in message` if false.